web/gui/README.md

```diff
@@ -9,6 +9,9 @@ learn_rel_path: "Operations"
 
 # Local Agent dashboard
 
+> ⚠️ You're checking the documentation for the previous Agent dashboard. For the current version please check [Accessing Netdata Dashboards](https://github.com/netdata/netdata/blob/master/docs/category-overview-pages/accessing-netdata-dashboards.md).
```

Comment:
Should we refer to the old dashboard as e.g. `legacy` ? That will provide a clear distinction between the old and the new one maybe.